itunes_app_scraper/scraper.py

```diff
@@ -266,6 +274,8 @@ def get_app_ratings(self, app_id, countries=None):
 
 			try:
 				result = requests.get(url, headers=headers).text
+				if sleep is not None:
```

Comment:
the only thing that matches the current architecture of library would be a 
`time.sleep(1)`
before the get request - no parameter

Reply:
Well, I think we still could keep the sleep time parametrized to allow flexible times. E.g. if scripts might run in parallel the user might want to increase the sleep time per request.